### DISCUSSION TITLE
[Feature] support internal and external URLs

### DISCUSSION THREAD

**Discussion by STIMECE** [immich-app/immich#2254] [Feature Request]
### The feature

This idea I got from home assistant. If I am home on my wifi, immich can see that I am connected to that network and if I am then it should acres my local address of immich and upload photos that way only....then if I am not home use the external address and add an option to manually upload a batch of photos.

I have data limits on my services where uploading a huge chunk of photos eats up.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by Allram** [comment]
This you can control on your Reverse Proxy and your router.
Don't think it's anything Immich can control.

**Reply by STIMECE** [reply]
Hmmm interesting I will look that up. I have a reverse proxy setup already so I will give it a shot.

**Comment by bo0tzz** [comment]
Like @Allram mentioned, this isn't in the scope of Immich. It's also a duplicate of #308.
